docs/install/install-from-package/os-preparation.rst

```diff
@@ -113,11 +113,42 @@ information on the Python version compatibility can be found at
 `here <https://github.com/lablup/backend.ai#package-installation-guide#python-version-compatibility>`_.
 
 There can be several ways to prepare a specific Python version. Here, we will be
-using pyenv and pyenv-virtualenv.
+using a standalone static built Python.
 
+Use a standalone static built Python (Recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Obtain distribution of `a standalone static built Python <https://github.com/indygreg/python-build-standalone/releases>`_ according to required 
+python version, target machine architecture and etc. Then extract the distribution 
+to a directory of your choice.
+
+.. code-block:: console
+
+   $ curl "https://github.com/indygreg/python-build-standalone/releases/download/${PYTHON_RELEASE_DATE}/cpython-${PYTHON_VERSION}+${PYTHON_RELEASE_DATE}-${TARGET_MACHINE_ARCHITECTURE}-${ARCHIVE_FLAVOR}.tar.gz > cpython-${PYTHON_VERSION}+${PYTHON_RELEASE_DATE}-${TARGET_MACHINE_ARCHITECTURE}-${ARCHIVE_FLAVOR}.tar.gz"
+   $ tar -xf "cpython-${PYTHON_VERSION}+${PYTHON_RELEASE_DATE}-${TARGET_MACHINE_ARCHITECTURE}-${ARCHIVE_FLAVOR}.tar.gz"
+   $ mkdir -p "/home/${USERNAME}/.static-python/versions"
+   $ mv python "/home/${USERNAME}/.static-python/versions/${PYTHON_VERSION}"
+
+For example,
+
+.. code-block:: console
+
+   $ curl "https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.11.6+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz > cpython-3.11.6+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz"
+   $ tar -xf "cpython-3.11.6+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz"
+   $ mkdir -p "/home/bai/.static-python/versions"
+   $ mv python "/home/bai/.static-python/versions/3.11.6"
+
+Update your PATH depending on your shell, restart your shell for the PATH changes to take effect.
 
-Use pyenv to manually build and select a specific Python version
+.. code-block:: console
+
+   $ echo 'export PATH=".static-python/versions/${PYTHON_VERSION}/bin:$PATH"' >> ~/.bashrc
```

Comment:
I'm not sure about setting the statically built Python as the default for the user's shell. Instead, I suggest utilizing a script, such as the one found at https://github.com/lablup/backend.ai-ansible/blob/main/remotes/home/scripts/autoenv.sh, to activate the per-directory Python environment.

Reply:
I will look more how to set Static Python and write more documentation setting env using venv.

Reply:
modifying systemd runner script using venv